src/components/Main.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { TodoList } from './TodoList';
+
+export const Main: React.FC = () => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
you can make this section part of `TodoList` component (instead of div wrapper there) and get rid of this component